metricbeat/module/mongodb/collstats/data.go

```diff
@@ -0,0 +1,62 @@
+package collstats
+
+import (
+	"strings"
+
+	"github.com/elastic/beats/libbeat/common"
+)
+
+func eventMapping(key string, data common.MapStr) common.MapStr {
+	names := strings.SplitN(key, ".", 2)
+
+	event := common.MapStr{
+		"db":         names[0],
+		"collection": names[1],
+		"name":       key,
+		"statistics": common.MapStr{
```

Comment:
Do we need here the `statistics` namespace? As this is already under `mongodb.collstats` I think we can put all the stats below directly on the same level.